lib/internal/test_runner/test.js

```diff
@@ -579,6 +579,8 @@ class Test extends AsyncResource {
     this.activeSubtests = 0;
     this.pendingSubtests = [];
     this.readySubtests = new SafeMap();
+    this.unfinishedSubtests = new SafeSet();
```

Comment:
nit: I think there is some overlap in meaning between `pendingSubtests` and `unfinishedSubtests` which makes the purpose of this property less clear

Reply:
Pending subtests are ones that have been queued to execute but need to wait due to concurrency. Unfinished means they have not finished executing.